proto/iot.proto

```diff
@@ -119,6 +126,34 @@ message Thermostat {
     Double cooling_setpoint = 11;
 }
 
+//Schedule of one week
+message Schedule {
+    //Map day of week to schedule
+    map<string, ScheduleDay> scheduleMap = 1;
+}
+
+// Repeated Schedule Wrapper
+message ScheduleDay {
+    //Series of Schedule Blocks
+    repeated ScheduleBlock blocks = 1;
+}
+
+// Schedule Block
+message ScheduleBlock {
+    //heating setpoint
+    //unit: celsius
+    Double heat_setting = 1;
```

Comment:
Can we unify the terminology with the thermostat definition? `heating_setpoint` and `cooling_setpoint`

Reply:
Gotchu, changed to the above :)